lib/Sema/TypeCheckStorage.cpp

```diff
@@ -2491,6 +2492,7 @@ SynthesizeAccessorRequest::evaluate(Evaluator &evaluator,
 
   switch (kind) {
   case AccessorKind::Get:
+  case AccessorKind::DistributedGet:
```

Comment:
Same here - we already have a decl from this which is produced by `getDistributedThunk()` right?